server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -9,7 +9,7 @@ import { fromUtf8ToBase64 } from "@fluidframework/common-utils";
 import * as git from "@fluidframework/gitresources";
 import { IClient, IClientJoin, ScopeType } from "@fluidframework/protocol-definitions";
 import * as core from "@fluidframework/server-services-core";
-import { validateTokenClaims } from "@fluidframework/server-services-client";
+import { validateTokenClaims } from "@fluidframework/server-services-utils";
```

Comment:
For example, this package can use server-services-utils cause its not isomorphic and only meant to run on the server